.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish to GitHub / Pypi
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest pytest-asyncio
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f ./tests/requirements.txt ]; then pip install -r ./tests/requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: pytest
+
+  build_and_publish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest pytest-asyncio twine
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f ./tests/requirements.txt ]; then pip install -r ./tests/requirements.txt; fi
```

Comment:
You don't need the following in the publish phase: 
- `flake8 pytest pytest-asyncio`
- you don't need `./tests/requirements.txt`

You do need the `wheel` package, otherwise twine will upload only `egg`.